npc/dev/test.txt

```diff
@@ -729,6 +729,29 @@ function	script	HerculesSelfTestHelper	{
 	callsub(OnCheckStr, "sprintf (positional)", sprintf("'%2$+05d'", 5, 6), "'+0006'");
 	callsub(OnCheckStr, "sprintf (positional)", sprintf("'%2$s' '%1$c'", "First", "Second"), "'Second' 'F'");
 
+	// unit data test
+	freeloop(1);
```

Comment:
I have reverted the NPC change. We can increase the check_gotocount in a separate commit and re-add this testing if necessary. Since most of the functionality of the `HerculesSelfTestHelper` is related to scripting language tests, this one seems unfit to be part of it as well.